<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectWidget</class>
 <widget class="QWidget" name="SelectWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>688</width>
    <height>464</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>选择学生信息</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="idlabel">
       <property name="text">
        <string>学号</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="namelabel">
       <property name="text">
        <string>姓名</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="mathlabel">
       <property name="text">
        <string>数学成绩</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="enlabel">
       <property name="text">
        <string>英语成绩</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="clabel">
       <property name="text">
        <string>c语言成绩</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="avglabel">
       <property name="text">
        <string>平均成绩</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>性别</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label1">
       <property name="text">
        <string>地址</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>邮箱</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="phonelabel">
       <property name="text">
        <string>电话</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>年龄</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="idLineEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="showNameLabel">
         <property name="text">
          <string>NULL</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="showMathLabel">
         <property name="text">
          <string>NULL</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="showEnLabel">
         <property name="text">
          <string>NULL</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="showCLabel">
         <property name="text">
          <string>NULL</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="showAvgLabel">
         <property name="text">
          <string>NULL</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="showSexLabel">
         <property name="text">
          <string>NULL</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="showLocationLabel">
         <property name="text">
          <string>NULL</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="showEmailLabel">
         <property name="text">
          <string>NULL</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="showPhoneLabel">
         <property name="text">
          <string>NULL</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="showAgeLabel">
         <property name="text">
          <string>NULL</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="0" column="2">
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <widget class="QPushButton" name="selectButton">
       <property name="text">
        <string>查询</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="returnButton">
       <property name="text">
        <string>主菜单</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
